R/listing_export.R

```diff
@@ -10,6 +10,6 @@
 #'   add_listing_col("AESER / AREL", fun = function(df) paste(df$AESER, df$AREL, sep = " / "))
 #' main_title(lsting) <- "this is some title"
 #' main_footer(lsting) <- "this is some footer"
-#' cat(export_as_txt(lsting, file = NULL, paginate = TRUE))
+#' cat(export_as_txt(lsting, file = NULL, paginate = TRUE, rep_cols = length(get_keycols(lsting))))
```

Comment:
Should be this automatic?

Reply:
I thought of changing this in the export_as_txt function but right now it uses rep_cols=num_cols. The method doesn't look like it's capturing the key_cols correctly.

Reply:
I think we may need to change that. I do not know. @shajoezhu @gmbecker ?